### PR TITLE
[TTAHUB-11] Add Next Steps Complete Date

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Add a complete date to next steps.

**Note:** This will be merged into the new **main-ar-redesign** branch once its passing the build. For now I think we can begin the code review process.

## How to test

Create a new report. On the 'Next Steps' page Create / Add / Remove / Update new steps.

Ultimately you should be able to create and submit and approve a report containing Next Steps with dates.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-11

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Comment by jasalisbury** [comment]
Is there a link to the design for this ticket?

**Comment by jasalisbury** [comment]
User should be able to select a date in the future. Actually I _could_ see us not allowing the user to select a date in the past, since these are the "next" steps.

**Comment by jasalisbury** [comment]
(sorry for third comment) Do we want to show the estimated completion date on the `review and submit` page?

File: frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js [hunk 1]
```diff
@@ -76,68 +93,124 @@ export default function NextStepsRepeater({
...
+              key={`${stepType}-next-step-form-group-step-${index + 1}`}
+              className="margin-top-2 margin-bottom-2"
+              error={blurStepValidations[index] || (errors[name] && errors[name][index]
+                && errors[name][index].note)}
             >
-              <Label
-                htmlFor={`${name === 'specialistNextSteps'
-                  ? 'specialist' : 'recipient'}-next-step-${index + 1}`}
-                className="sr-only"
+              <legend>
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js:109]: - You can remove the legends here, they only belong in ```<fieldset>```.

- I think the labels should just reflect the actual text you see. (When do you anticipate completing this step?) We should consult the design team and ask if we can change that to When do you anticipate completing step 1?, etc.

- it looks like the colon is in the wrong place - it should be "Step 1**:** *" not "Step**:** 1 *"

**Comment by thewatermethod** [comment]
Can main be merged into this branch, or the target branch? I can't test the backend stuff without switching node environments

**Comment by AdamAdHocTeam** [comment]
> Is there a link to the design for this ticket?

I think its more or less this: https://www.figma.com/proto/rJGMygyqMz8jjCcoEpzcnK/Activity-Reports?page-id=2627%3A135469&node-id=3261%3A120890&viewport=-3987%2C-5417%2C0.5&scaling=min-zoom&starting-point-node-id=3261%3A120890&show-proto-sidebar=1

**Pull request closed without merging**